backend/Dockerfile.schemaspy

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11 AS build
+WORKDIR /home/schemaspy
+
+RUN apt update && apt install -y graphviz
+RUN curl -H "Accept: application/zip" -L https://github.com/schemaspy/schemaspy/releases/download/v6.1.0/schemaspy-6.1.0.jar > schemaspy.jar
+RUN curl -H "Accept: application/zip" https://jdbc.postgresql.org/download/postgresql-42.2.18.jar > postgresql.jar
+
+RUN java -jar schemaspy.jar -debug -t pgsql -db simple_report -u postgres -p admin_password_for_local_dev_is_not_very_secure -host localhost:5432 -o output -s "simple_report" -dp postgresql.jar
```

Comment:
Maybe inject SR_DB_PORT here? Or actually, have it hit the DB over the docker network?

Reply:
Created a named network and set up schemaspy to access the db through it 😄